documentation/docs/14-configuration.md

```diff
@@ -14,7 +14,7 @@ const config = {
 	extensions: ['.svelte'],
 
 	kit: {
-		adapter: null,
+		adapter: { adapt: () => {} },
```

Comment:
this is not the actual default value, so it's wrong to say this

Reply:
but it can't be null as well, because it throws error,... ahhh. this is confusing.

Reply:
The default is no adapter. You must provide one to be able to build

Reply:
but what `no adapter` means in code,... I think, if I copy "default config" from documentation, I expect it will work, and not throw errors.